src/test/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollectionUnitTest.java

```diff
@@ -110,6 +112,77 @@ public void testIntervalMergingRuleAdjacentNoMerge(IntervalArgumentCollection ia
+    public void testIntervalUnmergingWithUnmappedIntervalsAndPadding(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:100-200");
+        iac.addToIntervalStrings("UNMAPPED");
+        iac.intervalPadding = 50;
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).size(), 1);
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 50, 250));
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalUnmergingWithIntervalPaddingOverlap(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:100-200");
+        iac.addToIntervalStrings("1:250-350");
+        iac.intervalPadding = 50;
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 50, 250));
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 200, 400));
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleOverlappingNoMergeWithPadding(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.intervalPadding = 10;
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 1, 110));
+        Assert.assertEquals(iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 41, 210));
+    }
+
+
+    @DataProvider(name = "NoMergeIllegalArguments")
+    public Object[][] NoMergeIllegalArguments(){
+        List<Object[]> argumentCollections = new ArrayList<>();
+
+        IntervalArgumentCollection iacXL = new RequiredIntervalArgumentCollection();
+        iacXL.excludeIntervalStrings.add("1:1-100");
+        argumentCollections.add(new Object[]{iacXL});
+
+        IntervalArgumentCollection iacMergingRule = new RequiredIntervalArgumentCollection();
+        iacMergingRule.intervalSetRule = IntervalSetRule.INTERSECTION;
+        argumentCollections.add(new Object[]{iacMergingRule});
+
+        return argumentCollections.toArray(new Object[][]{});
+    }
+
+    @Test(dataProvider = "NoMergeIllegalArguments", expectedExceptions = UserException.class)
+    public void testIntervalMergingRuleNoMergeIllegalArgumentCombinations(IntervalArgumentCollection iac) {
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.getIntervalsWithoutMerging(hg19GenomeLocParser.getSequenceDictionary());
+    }
```

Comment:
I don't think this test case is valid in the latest version of the branch, as the method no longer checks for INTERSECTION / XL as error conditions. Recommend that you remove it, and replace with a test that checks the "no inclusion intervals specified" case.